packages/browser-repl/src/components/editor.tsx

```diff
@@ -1,10 +1,9 @@
 import React, { Component } from 'react';
-import AceEditor from 'react-ace';
-import type { IAceEditor } from 'react-ace/lib/types';
 import 'ace-builds';
-import 'ace-builds/webpack-resolver';
```

Comment:
I was barking up the wrong tree - webpack-resolver is a bit of a sledgehammer here and what compass does is it uses AceEditor with setOptions useWorker: false so loading the worker files isn't an issue in that case anyway.